src/loaders/svg/tvgXmlParser.h

```diff
@@ -46,10 +46,11 @@ enum class SimpleXMLType
 
 typedef bool (*simpleXMLCb)(void* data, SimpleXMLType type, const char* content, unsigned length);
 typedef bool (*simpleXMLAttributeCb)(void* data, const char* key, const char* value);
+typedef bool (*simpleXMLW3CAttributeCb)(void* data, const char* key, const char* value, bool style);
 
 bool simpleXmlParseAttributes(const char* buf, unsigned buflen, simpleXMLAttributeCb func, const void* data);
 bool simpleXmlParse(const char* buf, unsigned buflen, bool strip, simpleXMLCb func, const void* data);
-bool simpleXmlParseW3CAttribute(const char* buf, simpleXMLAttributeCb func, const void* data);
+bool simpleXmlParseW3CAttribute(const char* buf, simpleXMLW3CAttributeCb func, const void* data);
```

Comment:
I don't think the `W3C` keyword has anything to do with the priority of the style attribute.
And I don't think we need to create a new function definition.

`_parseStyleAttr(void*, char*, char*)` calls `_parseStyleAttr(data, key, value, true);` and 
Calling each `_parseStyleAttr(... , false)`, we can avoid unnecessary definitions.

+) If we allow a little more code duplication, we can reduce the number of if-checks when looking for styleTags by making style=true and style=false functions respectively. It is optional anyway.